src/server/plugins/engine/views/components/list.html

```diff
@@ -1,13 +1,27 @@
-{% macro List(params) %}
-    {% if params.model.type == 'numbered' %}
-        <ol class="govuk-list govuk-list--number {% if params.classes %} {{ params.classes }} {% endif %}"{% for attribute, value in params.attributes %} {{attribute}}="{{value}}"{% endfor %}>
+{% macro List(component) %}
```

Comment:
Indentation seems a little off in this file

Reply:
Yeah, I decided to leave it (incorrectly) 4-space indented for now

Job for another PR

Glad you spotted it too